src/ledgerapp.ts

```diff
@@ -171,15 +167,14 @@ export class LedgerApp {
       const errorCodeData = response.slice(-2);
       const errorCode = errorCodeData[0] * 256 + errorCodeData[1];
 
-      return {
+      const success: LedgerAppVersion = {
         testMode: response[0] !== 0,
-        major: response[1],
-        minor: response[2],
-        patch: response[3],
+        version: `${response[1]}.${response[2]}.${response[3]}`,
```

Comment:
Hopefully we never need 256 or more patch releases for a given minor version!

Reply:
Interesting point. We can only do 16777216 releases, so we better be careful with those.